spec/fixtures/course/programming_single_test_suite_report_fail_eval.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Consolidate this file with `programming_single_test_suite_report.xml`.

I think you can adjust the tests so only 1 report fixture is required.

You can either adjust the tests in this PR to look for other test types failing, or check the existing tests that use `programming_single_test_suite_report.xml` and see if they even care about which type of test fails.

Reply:
`programming_single_test_suite_report.xml` has one skipped test case as well.

This will need a bit of digging, since the file is used in quite a few places. For now, i'm not sure about the reason for the skipped test case.